healthstatusprovider.go

```diff
@@ -0,0 +1,166 @@
+package probe
```

Comment:
Shall we follow some repo layout, e.g. https://github.com/golang-standards/project-layout ?

Reply:
good idea 👍